### PR TITLE
add prod in instance_method

### PR DESCRIPTION
**Pull request by AmeenUrRehman**
- [ ] #5713

### CODE REVIEW

**Comment by AmeenUrRehman** [comment]
@sharique1006 plz check could we are ready to merge !!!

**Comment by AmeenUrRehman** [comment]
@juliagsy , hey could we are ready to merge!!!

**Comment by juliagsy** [comment]
I'll review asap! thanks!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -364,3 +364,61 @@ def test_numpy_instance_argsort(
...
+    keepdims,
+    initial,
+    where,
+    as_variable,
+    num_positional_args,
+    native_array,
+):
+    input_dtype, x, axis = dtype_x_axis
+    where, as_variable, native_array = np_frontend_helpers.handle_where_and_array_bools(
+        dtype=dtype, out=out, keepdims=keepdims, initial=initial,
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:395]: please remove this line, the arguments below this line are the only needed ones

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hello! Great work so far! Just one small change and you're almost good to go! Please also ensure that the tests are passing! As usual, please request my review when you're done! Thanks!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py (hunk 1 above)
Comment (AmeenUrRehman) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py]: @juliagsy done plz check this!!!

**Pull request closed without merging**